git-versions.yml

```diff
@@ -64,8 +64,5 @@ services:
     monitor-client:
         image: mendersoftware/monitor-client:master
 
-    mender-gateway:
-        image: mendersoftware/mender-gateway:master
-
```

Comment:
as a review I checked that the definition exists in `./git-versions-enterprise.yml`:
```yaml
    mender-gateway:
        image: registry.mender.io/mendersoftware/mender-gateway:master
```

was it ever present in the `mendersoftware/mender-gateway:master`? (it is not at the moment)

Reply:
I am not sure I understand your question. Are you asking about the Docker container registry?

The name of the image was not really used (because mender-gateway was not a Docker image at the time) and it was originally wrongly added to the OS git versions file instead of the enterprise. It only matters now that we are publishing the Docker image